zebrad/src/components/mempool.rs

```diff
@@ -224,7 +234,9 @@ impl Service<Request> for Mempool {
                     if let Ok(tx) = r {
                         // Storage handles conflicting transactions or a full mempool internally,
                         // so just ignore the storage result here
-                        let _ = storage.insert(tx);
+                        let _ = storage.insert(tx.clone());
+                        // Send the transaction to peers
+                        let _ = self.transaction_sender.send(Some(tx.id))?;
```

Comment:
Efficiency: Can we gossip all the accepted transactions in the same peer request?

That way, we only send one network message to each peer, even if we just verified lots of transactions. (If we gossip one message per transaction per peer, that could be 1000 messages if we have 10 transactions and 100 peers.)